docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,107 @@
+# Move an instance from one Cleura Cloud region to another
+
+This guide will show you how to move or copy your instance to a different region in Cleura cloud.
```

Comment:
Wherever you've been using the string `Cleura Cloud`, use `{{extra.brand}} Cloud` instead.